checks/evaluation/sast.go

```diff
@@ -75,6 +79,11 @@ func SAST(name string,
 		// retun checker.InconclusiveResultScore.
 		return checker.CreateRuntimeErrorResult(name, sce.ErrScorecardInternal)
 	}
+	if snykScore != checker.InconclusiveResultScore {
+		if snykScore == checker.MaxResultScore {
```

Comment:
the first if statement is redundant, this can just be:
```go
if snykScore == checker.MaxResultScore {
	return checker.CreateMaxScoreResult(name, "SAST tool detected: Snyk")
}